### DISCUSSION TITLE
【委托】我这是在哪？

### DISCUSSION THREAD

**Discussion by bxzwq** [share121/inter-knot#5801] [公告]
![Uploading IMG_20241222_115636.png…]()
我正在光映广场上人行道散步呢，一辆失控的货车朝我撞过来，我当时以为自己死定了，结果醒来时就发现我在沙滩边醒来了，不过这给我干哪来了，这还是新艾利都吗，有没有绳匠能给我接回去的？

**Comment by bxzwq** [comment]
![IMG_20241222_115636](https://github.com/user-attachments/assets/27ea2875-5e52-46db-90f7-f181ad927b6b)
补一张图

**Reply by BlueWhale888** [reply]
这题我会，你要找到一个白毛小飞飞，然后游历七国，最后决战天理，打破虚假之天，最后让路过的星穹列车带你回家

**Comment by ConsistDust3838** [comment]
没见过啊，这奇怪的空洞 居然还有这么大片海

**Comment by 76asurname76** [comment]
会不会是老厄匹斯港，掉进空洞那么久，被以骸破坏成这样倒也不是不可能。

**Comment by LuYaoSEN** [comment]
这里我去过，据我了解这个叫提瓦特的空洞它的面积极其的大，大的非常离谱，想要出去可能比较难，但你可以找里面的神来帮助你

**Comment by he222111** [comment]
这个委托我接了。
